02-Logging/01-Displaying-Status-Information.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>I'm not sure exactly which intro notebook you mean here (I assume the index for Introducing Molecular Dynamics), but it should be linked.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I can try to link this, but I have had trouble with <code>nbsphinx</code>  handling relative links between notebooks like this in the past.</p>

Reply:
Got it, I would give it a shot but not a huge loss if it doesn't quite work. In that case definitely make sure that the names line up, though ("Introduction to..." vs "Introducing..."), otherwise someone somewhere will get confused.

Reply:
<p>Yes, I fixed that as well.</p>